Core/Main/elxMainExeUtilities.cxx

```diff
@@ -108,7 +112,7 @@ elastix::GetExtendedVersionInformation(const char * const executableName, const
 #  endif
 #endif
                      << newline << "Memory address size: " << std::numeric_limits<std::size_t>::digits << "-bit"
-                     << newline << "CMake version: " ELX_CMAKE_VERSION << newline
+                     << newline << "CMake version: " << ELX_CMAKE_VERSION << newline
```

Comment:
Please revert this change, if it is unnecessary. The preprocessor should still be able to do string concatenation, right?

Reply:
Do I understand correctly that you do not use CMake, when building elastix for Wasm?

Reply:
It does use CMake, but the preprocessor variable is not set, so a fallback is provided.

Reply:
Does it process elastix\Core\CMakeLists.txt ?

Reply:
Yes.

In the most recent commit, I removed the addition of `<<` here.